course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+| 5    | Sep 26 | Qualitaite review of population models              | Madeleine      |
+| 6    | Sep 28 | Qualitative population models in 2 dimensions       | Madeleine      |
+| 7    | ...    | Fixed points, null clines, and stability            | Madeleine      |
+| 8    |        | Modeling in R                                       | Madeleine      |
+| 9    |        | Fitting models to data                              | Madeleine      |
+| 10   |        | Linear regression population dynamics models?       | Luke           |
+| 11   |        | Introduce data sets, getting started with projects  | Luke           |
+| 12   |        | Project work, prodigenR, tidy data, git via rstudio | Luke           |
+| 13   |        | Data visualization in R                             | Lindsay        |
+| 14   |        | Project work, data visualizations                   | Lindsay        |
+| 15   |        | Reproducible workflow, metadata, Rmarkdown          | Lina or Elliot |
+| 16   |        | Project work                                        | 1 TA/group     |
+| 17   |        | Scientific collaboration, Git and GitHub            | Elliot         |
+| 18   |        | Project work                                        | 1 TA/group     |
+| 19   |        | Preparing a scientific item for academic publishing | Lina           |
+| 20   |        | Project work                                        | 1 TA/group     |
+| 21   |        | Project work                                        | 1 TA/group     |
+| 22   |        | Project work                                        | 1 TA/group     |
+| 23   |        | Project work                                        | Everyone       |
+| 24   |        | Project work                                        | Everyone       |
+| 25   |        | ??? We only included 24 sessions                    |                |
+
+# Assessment schedule 
+| Assesment             | Type              | Due date         | Marks |
+|-----------------------|-------------------|------------------|-------|
+| BasicR                | Individual        | Tue weeks after? | 10    |
+| dplyr                 | Individual        |                  | 10    |
+| Population models     | Individual        |                  | 10    |
+| Modeling in R         | Individual        |                  | 10    |
+| Linear regression?    | Individual        |                  | 10    |
+| Setup github          | Group/Individual? |                  | 10    |
+| Data visualization    | Group/Individual? |                  | 10    |
+| Reproducible workflow | Group/Individual? |                  | 10    |
+| Final project work    | Group             |                  | 20    |
+
+_There will be a penalty of -5% per day (including week-ends) for any late assignments._
+
+> Joel: Compare the above with the table below from Martin's example syllabus. In the above version 20-50% could be project work. Comments?
```

Comment:
I included the table below for comparison, will remove in the final version. They have 40 points on assignments and 60 on exams. We have 80 points on assignments and 20 on the final project. Feel free to suggestion redistributions, I just put this together for now.